### PR TITLE
Uses object detection in cloud vision api, updating requirements.txt, remove .pem files, venv files

### PR DESCRIPTION
**Pull request by vrajesh978**
Uses object detection in cloud vision api, 
updating requirements.txt, 
remove .pem files, venv files

### CODE REVIEW

**Comment by lipim12** [comment]
2543 files at once - woah!

File: user.py [hunk 1]
```diff
@@ -22,12 +23,12 @@ def get_id_user(id):
...
                         name=user['name'],
                         email=user['email'],
-                        profile_pic= user['profile_pic'],
+                        profile_pic=user['profile_pic'],
                         food_consumptions=user['food_consumptions'])
         return user_obj
 
     @staticmethod
-    def get(db,email):
+    def get(db, email):
```
Comment (lipim12) [inline user.py:31]: Using any specific code styling tool?

**Review by lipim12 — CHANGES_REQUESTED** [review]
Are all of the non `.py` files really required to be merged? Could you delete unnecessary files like `.pem` or add them to `.gitignore` for future commits?

File: user.py (hunk 1 above)
Comment (vrajesh978) [reply user.py:31]: No tool. It's just pep8 standards.

Comment (lipim12) [reply user.py:31]: I was asking if you changed everything manually? or used a style guide/sheet/tool for that?

Comment (vrajesh978) [reply user.py:31]: I am following pep8 standards. You can find this at (https://www.python.org/dev/peps/pep-0008/)

**Comment by vrajesh978** [comment]
@lipim12, this pr is for removing unnecessary files like .pem .sh venv folder.

File: user.py (hunk 1 above)
Comment (vrajesh978) [reply user.py:31]: I am using vscode

**Review by lipim12 — APPROVED** [review]

**Pull request merged**